pkg/reconciler/taskrun/dynamic.go

```diff
@@ -228,6 +228,6 @@ func (r DynamicResolver) Allocate(taskRun *ReconcileTaskRun, ctx context.Context
 		}
 	}
 
-	return reconcile.Result{}, nil
+	return reconcile.Result{RequeueAfter: 2 * time.Minute}, nil
```

Comment:
Can you expand on what this changes does. Does this means that we will wait for an extra 2 minutes before letting the provision task run?

Reply:
Hi, not exactly. 
If `requeue` is not set, that means next reconcillitation will happen _only_ when Taskrun will be modified.
That's why we're missing reaction to VM become available for the Taskun, because just nothing veriifies this fact, as nothing changes the TaskRun 
Setting this time out forces reconcile the object just by passing some time. 2 min is empiric estimated value which we may have found VM ready from the first try, and not too much overhead if not.

Reply:
thanks for the explanation